mteb/tasks/Classification/ara/AJGT.py

```diff
@@ -37,3 +38,38 @@ class AJGT(AbsTaskClassification):
 }
 """,
     )
+
+
+class AJGTV2(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="AJGT.v2",
+        dataset={
+            "path": "mteb/ajgt",
+            "revision": "6c3a9c1aaf2d9fd5e41508c33eb250cd6804ce98",
+        },
+        description="Arabic Jordanian General Tweets (AJGT) Corpus consisted of 1,800 tweets annotated as positive and negative. Modern Standard Arabic (MSA) or Jordanian dialect.",
```

Comment:
We need a description of the changes here as well, so that people know what the differences between the two are.